server/resolvers/review/query.js

```diff
@@ -0,0 +1,7 @@
+const { Review } = require("../../models/mongodb");
+
+const ReviewQuery = {
+  reviews: ({ book_id }) => Review.find({ book_id }),
+};
+
+module.exports = { ...ReviewQuery };
```

Comment:
`...` on a Pascal case seems odd to me 😄 why not have `const reviewQuery` ? 

And the question, when to use which case?

Reply:
I have used PascalCase for resolvers, but yes should maintain consistency everywhere.